test/v1/services/RetrieveCharitableGivingReliefServiceSpec.scala

```diff
@@ -103,7 +103,15 @@ class RetrieveCharitableGivingReliefServiceSpec extends ServiceSpec {
           ("NOT_FOUND_PERIOD", NotFoundError),
           ("NOT_FOUND_INCOME_SOURCE", NotFoundError),
           ("SERVER_ERROR", InternalError),
-          ("SERVICE_UNAVAILABLE", InternalError)
+          ("SERVICE_UNAVAILABLE", InternalError),
```

Comment:
This needs to follow the "extraTysErrors" pattern, to avoid it becoming a sort of "mega-blob" of errors.
e.g. see: https://github.com/hmrc/individuals-reliefs-api/blob/f641ce94a00f212d4b6c00ff282690d30e3d0c3f/test/v1/services/CreateAndAmendCharitableGivingTaxReliefServiceSpec.scala